### ISSUE TITLE
Cannot toggle pokemon on Firefox in the living dex

### ISSUE THREAD

**Issue by Statharas** [itsjavi/supereffective#107]
A recent change has broken the Living Dex on Firefox (have not tested other platforms) and clicking a pokemon does not do anything.

Attached is the living dex export, if that helps. Feel free to reach out to me on Discord.

Win11 22621
Firefox 116.0.3
[home---living-dex.livingdex.json.zip](https://github.com/itsjavi/supereffective/files/12485203/home---living-dex.livingdex.json.zip)

**Comment by Statharas** [comment]
No, it didn't throw anything. However, it is working fine as of now.

**Comment by Statharas** [comment]
Disregard everything I just said. Using the following options, I am able to replicate it, still
![image](https://github.com/itsjavi/supereffective/assets/2414202/689b983d-4bdd-47ab-a569-f88595910d09)

**Comment by Statharas** [comment]
Why would it make it read only? My boxes are already sorted that way :/

**Comment by matthew-long** [comment]
I think what happens is when you toggle "Shiny after non-shiny" it's changing the view, but the markers are happening on the boxes that would have been in that location if you had "Shiny after non-shiny" untoggled.
You can see this by toggling on "Shiny after non-shiny" and selecting Nidoqueen. Then untoggling "Shiny after non-shiny" and shiny bulbasaur will have the catch marker.
